crates/biome_cli/src/commands/clean.rs

```diff
@@ -1,10 +1,16 @@
-use crate::commands::daemon::biome_log_dir;
+use crate::commands::daemon::default_biome_log_dir;
 use crate::{CliDiagnostic, CliSession};
+use biome_flags::biome_env;
 use std::fs::{create_dir, remove_dir_all};
+use std::path::PathBuf;
 
 /// Runs the clean command
 pub fn clean(_cli_session: CliSession) -> Result<(), CliDiagnostic> {
-    let logs_dir = biome_log_dir();
+    let logs_dir = biome_env()
+        .biome_log_dir
+        .value()
+        .map(PathBuf::from)
+        .unwrap_or(default_biome_log_dir());
```

Comment:
For consistency, we could replace `_dir` with `_path`?
```suggestion
        .unwrap_or(default_biome_log_path());
```